hands-on-exercise/2-ignite-cli-adv/6-payment-winning.md

```diff
@@ -1041,4 +1041,4 @@ To summarize, this section has explored:
 
 <!--## Next up
 
-You can skip ahead and see how to integrate [foreign tokens](/hands-on-exercise/2-ignite-cli-adv/8-wager-denom.md) via the use of IBC, or take a look at the [next section](./6-gas-meter.md) to prevent spam and reward validators proportional to their effort in your checkers blockchain.-->
+You can skip ahead and see how to integrate [foreign tokens](../2-ignite-cli-adv/10-wager-denom.md) via the use of IBC, or take a look at the [next section](../2-ignite-cli-adv/8-gas-meter.md) to prevent spam and reward validators proportional to their effort in your checkers blockchain.-->
```

Comment:
```suggestion
You can skip ahead and see how to integrate [foreign tokens](./10-wager-denom.md) via the use of IBC, or take a look at the [next section](./8-gas-meter.md) to prevent spam and reward validators proportional to their effort in your checkers blockchain.-->
```